### PR TITLE
Add Dockerfile.init to export files from client db

### PR DESCRIPTION
**Pull request by svenbeck27**
Dockerfile.init spins up a **Firebird2.5** database server.
The database is mounted from a local directory.
**export.sh** will **dump** the **fbx data** of selected tables to the same mounted directory.

previous fbexport will still run, **makefile** is changed to support both **Ubuntu16.04 and 18.04**

**FBexport.cpp** is changed to include the follwoing PR:
https://github.com/mariuz/fbexport/pull/13

### CODE REVIEW

File: Dockerfile.init [hunk 1]
```diff
@@ -0,0 +1,22 @@
+FROM ubuntu:16.04
```
Comment (pedro-r-marques) [inline Dockerfile.init]: Maybe call this file Dockerfile.fb-2.5

File: Dockerfile [hunk 2]
```diff
@@ -6,7 +6,7 @@ RUN apt-get install -y g++ make libfbclient2
```
Comment (pedro-r-marques) [inline Dockerfile]: Maybe rename to Dockerfile.fb-3.0

File: export.sh [hunk 3]
```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+sed -i -e '/RemoteBindAddress =/ s/= .*/= /' /etc/firebird/2.5/firebird.conf
+
+fbguard -d
+
+TABLE_NAMES=(
+    "BEST"
```
Comment (pedro-r-marques) [inline export.sh]: sort alphabetically

File: fbexport/FBExport.cpp [hunk 4]
```diff
@@ -356,18 +357,25 @@ string FBExport::CreateHumanString(IBPP::Statement& st, int col)
     if (ar->ExportFormat == xefCSV)         // CVS format, each field's value is quoted
     {
         // escape quotes with two quotes " => ""
```
Comment (pedro-r-marques) [inline fbexport/FBExport.cpp]: The comment is now outdated, correct ?

File: README.md [hunk 5]
```diff
@@ -23,3 +23,10 @@ automatically loads tables and compares their fields
 uses Foreign Keys and Checks to determine the correct order of tables
 can create ALTER TABLE and CREATE TABLE scripts needed to update the destination database
 HTML overview of differences in data and metadata
+
+
+Export client data
+```
+docker build --build-arg DATEBASE_NAME_ARG=DATABASE_NAME.FDB -t firebirdexport -f Dockerfile.init .   
+docker run -v DATABASE_PATH:/var/lib/firebird/2.5/data -it firebirdexport
```
Comment (pedro-r-marques) [inline README.md]: The -it option is not required, correct ?

File: fbexport/FBExport.cpp (hunk 4 above)
Comment (svenbeck27) [reply fbexport/FBExport.cpp]: resolved.

**Pull request merged**